selfdrive/car/docs_definitions.py

```diff
@@ -269,6 +269,8 @@ def init(self, CP: car.CarParams, all_footnotes: Dict[Enum, int]):
       assert CP.minSteerSpeed == 0, f"{CP.carFingerprint}: Minimum steer speed set in both CarInfo and CarParams"
     else:
       self.min_steer_speed = CP.minSteerSpeed
+      if self.min_steer_speed < 1.0 * CV.MPH_TO_MS:
+        self.min_steer_speed = 0
```

Comment:
@sshane You had mentioned doing the rounding in `__post_init__` but this location feels logical to me. Let me know if you think it's a problem.